src/Core/SharedKernels/Questionnaire/WB.Core.SharedKernels.Questionnaire/Documents/Question/MultimediaQuestion.cs

```diff
@@ -20,19 +20,15 @@ public override void AddAnswer(Answer answer)
 
         public override IEnumerable<T> Find<T>(Func<T, bool> condition)
         {
-            return new T[0];

```

Comment:
need remove this method and need inhered this class from 'ExternalServiceQuestion'